docs/modules/ROOT/pages/scaling-upgrading.adoc

```diff
@@ -25,13 +39,25 @@ Assuming the following is the existing configuration:
 include::ROOT:example$/hazelcast-5.2.4.yaml[]
 ----
 
-and you want to upgrade your hazelcast version to {page-latest-supported-hazelcast}, you need to apply the following:
+and you want to upgrade your Hazelcast version to {page-latest-supported-hazelcast}, you need to apply the following:
 
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hazelcast.yaml[]
+----
 
-Assuming the following is the existing configuration:
+It is worth highlighting the following system properties, which are related to upgrades:
+
+- During a rolling upgrade, the cluster is shutdown gracefully to prevent data loss. You can configure this using the link:https://docs.hazelcast.com/hazelcast/5.3/system-properties#hazelcast.graceful.shutdown.max.wait[hazelcast.graceful.shutdown.max.wait system property].
+
+- The final step in the link:https://docs.hazelcast.com/hazelcast/5.3/maintain-cluster/rolling-upgrades#rolling-upgrade-procedure[rolling upgrade procedure] is to trigger a rolling upgrade on the cluster. Hazelcast Platform Operator triggers it automatically by setting `hazelcast.cluster.version.auto.upgrade.enabled` to `true` by default, and it cannot be changed.
+
+- `hazelcast.persistence.auto.cluster.state` set to `true` by default and `hazelcast.persistence.auto.cluster.state.strategy` is set to `NO_MIGRATION` by default. They cannot be changed.
```

Comment:
I think we need to move them to a separate section, as we discussed.

Reply:
Does it look good? https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/150/commits/90200db7368650511cb4056b0d055324ca87198b